src/OrchardCore.Modules/OrchardCore.OpenId/Startup.cs

```diff
@@ -6,6 +6,7 @@
 using Microsoft.AspNetCore.Authentication;
 using Microsoft.AspNetCore.Authentication.OpenIdConnect;
 using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Identity;
```

Comment:
Are these changes necessary now that you're no longer updating the identity options?

Reply:
no, removed